aiida/backends/utils.py

```diff
@@ -68,6 +71,29 @@ def get_settings_manager():
     return manager
 
 
+def validate_schema_generation():
+    """Verify that the database schema generation is compatible with the current code schema generation."""
+    from aiida.common.exceptions import ConfigurationError, NotExistent
+    try:
+        schema_generation_database = get_db_schema_generation().value
+    except NotExistent:
+        schema_generation_database = SCHEMA_GENERATION_VALUE
```

Comment:
I would have hardcoded `1` here - if in the future we reuse the code (I know we shouldn't but maybe someone tried to merge) and then up the SCHEMA_GENERATION_VALUE, still here the value should remain 1